Source/XUnitExtensions/Lit/OutputCheckCommand.cs

```diff
@@ -41,7 +41,7 @@ private abstract record CheckDirective(string File, int LineNumber) {
   }
 
   private record CheckRegexp(string File, int LineNumber, Regex Pattern) : CheckDirective(File, LineNumber) {
-    public static CheckDirective Parse(string file, int lineNumber, string arguments) {
+    public new static CheckDirective Parse(string file, int lineNumber, string arguments) {
```

Comment:
What's this syntax change doing? I don't know this C# feature.

Reply:
It's making the hiding of the base class's implementation explicit. ([docs](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/knowing-when-to-use-override-and-new-keywords))